ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Install pmm-dump, always overwrite the version before with latest
+
+- name: Download pmm-dump binary (tech preview)
+  get_url:
+    url: https://github.com/percona/pmm-dump/releases/download/0.5.1-beta/pmm-transferer
+    dest: /tmp/pmm-transferer
+    tmp_dest: /tmp/
+    force: yes
+    headers:
+      Accept: "application/octet-stream"
+
+- name: Display the File contents
+  shell: ls -laH /tmp/
```

Comment:
it's better to use `command` module instead `shell` when it's possible.

Reply:
Thanks! It's to debug, I'll remove it anyway.